### DISCUSSION TITLE
pnpm安装依赖报错

### DISCUSSION THREAD

**Discussion by harryhxs** [opentiny/tiny-engine#775] [Q&A]
![image](https://github.com/user-attachments/assets/5d0ac3bc-0041-45d5-920f-b1e5e899c0f7)
安装依赖的时候报错，有遇到这种情况吗？

**Comment by chilingling** [comment]
1. 无具体复现步骤以及信息，无法复现
2. 报错信息看不出来是什么错误，建议您查看完整错误信息重试。或给出更详细报错信息。

**Comment by liaodalin19903** [comment]
可以复制一下所有的报错信息，我们帮您检验。
